sphinx/templates/latex/content.tex_t

```diff
@@ -1,12 +1,23 @@
 %% Generated by Sphinx.
 \def\sphinxdocclass{<%= docclass %>}
+<% if latex_engine == 'lualatex' -%>
+\IfFileExists{luatex85.sty}
+ {\RequirePackage{luatex85}}
+ {\ifdefined\luatexversion\ifnum\luatexversion>84\relax
+  \PackageError{sphinx}
+  {** With this LuaTeX (\the\luatexversion),Sphinx requires luatex85.sty **}
+  {** Add the LaTeX package luatex85 to your TeX installation, and try again **}
+  \endinput\fi\fi}
+<% endif -%>
 \documentclass[<%= papersize %>,<%= pointsize %><%= classoptions %>]{<%= wrapperclass %>}
 \ifdefined\pdfpxdimen
    \let\sphinxpxdimen\pdfpxdimen\else\newdimen\sphinxpxdimen
 \fi \sphinxpxdimen=<%= pxunit %>\relax
 <%= passoptionstopackages %>
 <%= geometry %>
+<% if latex_engine == 'pdflatex' -%>
```

Comment:
This means `inputenc` key effects only for pdflatex. It's reasonable.
But document says:
>“inputenc” package inclusion, defaults to '\\usepackage[utf8]{inputenc}' when using pdflatex. **Otherwise** unset.

So we need to update the document (the "Otherwise" case).

BTW, what case will users update this key? I can't imagine that. If none, it would be better to deprecate the meanlingless key.